src/order/category/LinearOrder.lean

```diff
@@ -21,11 +27,38 @@ attribute [derive [large_category, concrete_category]] LinearOrder
 
 instance : has_coe_to_sort LinearOrder Type* := bundled.has_coe_to_sort
 
-/-- Construct a bundled LinearOrder from the underlying type and typeclass. -/
+/-- Construct a bundled `LinearOrder` from the underlying type and typeclass. -/
 def of (α : Type*) [linear_order α] : LinearOrder := bundled.of α
 
 instance : inhabited LinearOrder := ⟨of punit⟩
 
 instance (α : LinearOrder) : linear_order α := α.str
 
+instance has_forget_to_PartialOrder : has_forget₂ LinearOrder PartialOrder :=
+bundled_hom.forget₂ _ _
+
+/-- Constructs an equivalence between linear orders from an order isomorphism between them. -/
+@[simps] def iso.mk {α β : LinearOrder.{u}} (e : α ≃o β) : α ≅ β :=
```

Comment:
Perhaps `order_equiv.to_iso` would be nice as dot notation for this?

Reply:
Bhavik, this PR introduces four declarations called `iso.mk`. How the hell is that supposed to work with dot notation? 🤣

Reply:
`order_equiv.to_LinearOrder_iso`? I'm not really seeing the issue here!

Reply:
Alright, you may keep it as it is then!